src/forms/button/Button.stories.tsx

```diff
@@ -125,4 +125,14 @@ storiesOf("Button", module)
             </Button>
             <Button key="link" primary state={ButtonState.SUCCESS} onClick={action("link click")} icon="link" />
         </div>
+    ))
+    .add("Simple Button and Icon button without btn and btn-icon class", () => (
```

Comment:
If the customBtn property is to just get the button working correctly in the Header component, we can remove this as it doesn't really showcase useful functionality on its own.  It will be better suited in the Header storybook to show this property.

Reply:
As discussed in meeting, we are keeping this story in button stories.